patches/chrome-android-java-src-org-chromium-chrome-browser-settings-privacy-ClearBrowsingDataFragment.java.patch

```diff
@@ -1,46 +0,0 @@
-diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/privacy/ClearBrowsingDataFragment.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/privacy/ClearBrowsingDataFragment.java
```

Comment:
`Clear downloads` option was removed as it didn't work as expected. If we want to get it back we will use `changeMethodOwner` next time. For now I needed to revert these changes to make it compilable after adding `//chrome/android:chrome_test_java` dependency.